### PR TITLE
feat: add transfer manager MPU sample, adjust defaults, prepare for GA

### PR DESCRIPTION
**Pull request by ddelgrosso1**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/nodejs-storage/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: samples/README.md
Comment (danielduhh) [inline samples/README.md:131]:
```suggestion
  * [Upload a File in Chunks Utilzing Transfer Manager](#upload-a-file-in-chunks-with-transfer-manager)
```
for consistency

File: samples/README.md
Comment (danielduhh) [inline samples/README.md:2090]:
```suggestion
### Upload a File in Chunks With Transfer Manager
```

File: src/transfer-manager.ts
Comment (danielduhh) [inline src/transfer-manager.ts:36]: Maybe update the PR description to say you're taking off the training wheels?

Comment (danielduhh) [reply src/transfer-manager.ts:36]: Or split the PRs

**Comment by ddelgrosso1** [comment]
Marking `do not merge` until we have final approval to go ahead with launch. Please review!

**Review by danielduhh — APPROVED** [review]

**Review by danielbankhead — APPROVED** [review]

**Pull request merged**
